app/models/concerns/foreman_openscap/host_extensions.rb

```diff
@@ -46,6 +46,11 @@ module HostExtensions
       }
     end
 
+    def policies=(policies)
```

Comment:
in rails 4, you can't set an association which goes through more than one has_many through, which policies does, thus, we do what rails 3 used to do for us, and create the asset if it doesn't exist yet, then set the policies